### ISSUE TITLE
[TOTP] Downloaded and printed successful messages are displayed even if the processes are canceled

### ISSUE THREAD

**Issue by vasilicatamas** [mozilla/fxa#4029]
**Note:**
This is a follow-up issue for: #3877

**Build:**
Prod [Train 1.155.0]

**Affected platforms:**
Windows 10 64-bit

**Steps to reproduce:**
1. Launch Firefox and login using a FxA.
2. Navigate to Manage account page and click on "Enable" button from 2FA section.
3. Scan the QRcode and enter the provided key in the security code field.
4. Tap on Ready.
5. Select download/print option.
6. Cancel the download/print process.

**Expected results:**
“Downloaded” and “Printed” messages are displayed only after the download and print processes are successfully ended.

 **Actual Results:**
Successful messages are displayed just after the “Download” and “Print” buttons are selected.

![download](https://user-images.githubusercontent.com/18114210/73666905-2eb2a400-46ac-11ea-8f0a-e9e264391b0e.gif)

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1021)
┆Issue Number: FXA-1021

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

[~rfeeley@mozilla.com] - I don't think we can detect if that cancel button is hit.  Should we not show the success message at all?  Or any other changes you want to do here?

**Comment by data-sync-user** [comment]
➤ Ryan Feeley commented:

We could change the messages to Downloading… and Printing… to avoid over-promising. Definitely not worth the major engineering work to detect if the codes were fully downloaded/printed.

**Comment by data-sync-user** [comment]
➤ Jody Heavener commented:

No longer an issue due to Settings refresh.
